src/main/java/lotto/domain/Winners.java

```diff
@@ -0,0 +1,20 @@
+package lotto.domain;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public class Winners {
+	private static final int DEFAULT_VALUE = 0;
+	private static final int ADD_VALUE = 1;
+	private final Map<Integer, Integer> winners = new HashMap<>();
```

Comment:
```suggestion
	private final Map<Rank, Integer> winners = new EnumMap<>(Rank.class);
```

EnumMap을 홀용해볼 수도 있을 것 같아요 :)